_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -12,13 +12,23 @@ It provides a view on the *Typed Abstract Syntax Trees* and their properties suc
 ## How to use the API
 
 Accessing this API needs an import that depends the current `Quotes`.
```

Comment:
```suggestion
Accessing this API needs an import that depends on the current `Quotes`, which are implicitly in scope.
```

Reply:
Maybe this needs a bit more explanation.  This usage pattern is somewhat obvious to me, but it might not to newcomers. What about:

"The reflection API is defined in the type `Quotes`.  The actual instance depends on the current scope, in which quotes or quoted pattern matching is used. Hence, every macro method receives `Quotes` as an additional argument. Since `Quotes` is contextual, to access it members we either need to name the parameter, or summon it. The following definition of the standard library provides the canonical way of accessing it."